cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java

```diff
@@ -195,10 +195,22 @@ public void testSearchWithTags() throws Exception {
 
     // verify "*"
 
+    // components (services) are listed as optional, as metrics are cleared between tests,
+    // some of the services might not have emitter metrics yet since this test begining,
```

Comment:
"yet since this test beginning" - do you mean since the beginning of the test or since this test is just beginning?